src/Android/Xamarin.Android/Samples/GeometryEngine/Buffer/Buffer.cs

```diff
@@ -0,0 +1,149 @@
+                MapPoint theMapPoint = _myMapView.ScreenToLocation(e.Position);
+
+                // Get the buffer size from the UITextField.
+                double theBufferInMiles = System.Convert.ToDouble(_BufferDistanceEditText.Text);
+
+                // Create a variable to be the buffer size in meters. There are 1609.34 meters in one mile.
+                double theBufferInMeters = theBufferInMiles * 1609.34;
+
+                // Get a buffered polygon from the GeometryEngine Buffer operation centered on the map point. 
+                // Note: The input distance to the Buffer operation is in meters. This matches the backdrop 
+                // basemap units which is also meters.
+                Geometry theGeometryBuffer = GeometryEngine.Buffer(theMapPoint, theBufferInMeters);
+
+                // Create the outline (a simple line symbol) for the buffered polygon. It will be a solid, thick, green line.
+                SimpleLineSymbol theSimpleLineSymbol = new SimpleLineSymbol(SimpleLineSymbolStyle.Solid, System.Drawing.Color.Green, 5);
+
+                // Create the color that will be used for the fill of the buffered polygon. It will be a semi-transparent, green color.
+                System.Drawing.Color theFillColor = System.Drawing.Color.FromArgb(125, 0, 255, 0);
+
+                // Create simple fill symbol for the buffered polygon. It will be solid, semi-transparent, green fill with a solid, 
+                // thick, green outline.
+                SimpleFillSymbol theSimpleFillSymbol = new SimpleFillSymbol(SimpleFillSymbolStyle.Solid, theFillColor, theSimpleLineSymbol);
+
+                // Create a new graphic for the buffered polygon using the defined simple fill symbol.
+                Graphic thePolygonGraphic = new Graphic(theGeometryBuffer, theSimpleFillSymbol);
+
+                // Add the buffered polygon graphic to the graphic overlay.
+                _GraphicsOverlay.Graphics.Add(thePolygonGraphic);
+
+                // Create a simple marker symbol to display where the user tapped/clicked on the map. The marker symbol will be a 
+                // solid, red circle.
+                SimpleMarkerSymbol theSimpleMarkerSymbol = new SimpleMarkerSymbol(SimpleMarkerSymbolStyle.Circle, System.Drawing.Color.Red, 5);
+
+                // Create a new graphic for the spot where the user clicked on the map using the simple marker symbol. 
+                Graphic theUserInputGraphic = new Graphic(theMapPoint, theSimpleMarkerSymbol);
+
+                // Add the user tapped/clicked map point graphic to the graphic overlay.
+                _GraphicsOverlay.Graphics.Add(theUserInputGraphic);
+            }
+            catch (System.Exception ex)
+            {
+                // Display an error message if there is a problem generating the buffer polygon.
+                var alertBuilder = new AlertDialog.Builder(this);
```

Comment:
Let's be explicit about the type here, rather than using var, per the new code style.